File: beginner/1005/average_1.c
#include <stdio.h>

int	main(void)
{
	double	grade_1;
	double	grade_2;
	double	w1;
	double	w2;
	double	result;

	w1 = 3.5;
	w2 = 7.5;
	scanf("%lf %lf", &grade_1, &grade_2);
	result = ((grade_1 * w1) + (grade_2 * w2)) / (double)(w1 + w2);
	printf("MEDIA = %.5lf\n", result);
}

File: beginner/1005/Average1.java
// challenge link - https://judge.beecrowd.com/en/problems/view/1005

import java.io.IOException;
import java.util.Scanner;

public class Average1 {
	public static void main(String[] args) {
		Scanner reader = new Scanner(System.in);
		double weight1 = 3.5;
		double weight2 = 7.5;
		double grade1 = reader.nextDouble() * weight1;
		double grade2 = reader.nextDouble() * weight2;
		double average = (grade1 + grade2) / (weight1 + weight2);
		System.out.println("MEDIA = " + String.format("%.5f", average));
		reader.close();
	}
}
